backend/src/services/project/project-service.ts

```diff
@@ -841,14 +863,14 @@ export const projectServiceFactory = ({
       projectId = certManagerProjectFromSplit.id;
     }
 
-    const { permission, ForbidOnInvalidProjectType } = await permissionService.getProjectPermission(
+    const { permission } = await permissionService.getProjectPermission({
       actor,
       actorId,
       projectId,
       actorAuthMethod,
-      actorOrgId
-    );
-    ForbidOnInvalidProjectType(ProjectType.CertificateManager);
+      actorOrgId,
+      projectOperationType: ProjectOperationType.Global
```

Comment:
why does this say global when before it used to be for cert manager? some for the ones below

Reply:
Good catch. Forgot tony had cert and ssh in project service. Fixxed it